internal/driver/driver.go

```diff
@@ -545,17 +590,34 @@ func (s *Driver) getDBInfo(variable string) (dbInfo *DBInfo, err error) {
 		dbArray = strings.Split(variable, ".")
 		if len(dbArray) < 2 {
 			s.lc.Errorf("Db Area read variable should not be empty")
-			return
+			return nil, fmt.Errorf("DB variable %+v is invaliad", variable)
```

Comment:
invaliad? you should use `invalid`

Reply:
Please check for spelling errors in English.

Reply:
Okay, I have already checked it